docs(readme): add AwesomeTechStack to lighthouse integrations

AndreasKubasa opened the pull request:
**Summary**

AwesomeTechStack is for all developers and designers who want to improve their website or strive to get the best out of their new project.
With technology insights you will always know what is trending now and what would be the best tech stack to use at the moment

https://awesometechstack.com/tools/analyze/documentation/

AndreasKubasa commented:
@googlebot I signed it!

AndreasKubasa commented:
@benschwarz, is there something special we need to do to be mentioned @ the npms package Lighthouse Integrations part?

patrickhulce commented on readme.md:
```diff
@@ -345,6 +345,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Web Page Test](https://www.webpagetest.org)** — An [open source](https://github.com/WPO-Foundation/webpagetest) tool for measuring and analyzing the performance of web pages on real devices. Users can choose to produce a Lighthouse report alongside the analysis of WebPageTest results.
 
+* **[AwesomeTechStack](https://awesometechstack.com)** — AwesomeTechStack is a free to use website tech stack analyzer. AwesomeTechStack analyzes and rates the performance of technology stacks. AwesomeTechStack gives you insights about state-of-the-arts, up-to-dateness and secureness of any website's technology stack, with guidance to improve performance. Lighthouse insights are a crucial part of ...
```
this is an alphabetized list, so you can actually put this first :)

patrickhulce replied:
```suggestion
* **[AwesomeTechStack](https://awesometechstack.com)** — AwesomeTechStack is a free to use website tech stack analyzer. AwesomeTechStack provides insights into the security, modernity, and performance of any website's technology stack and guidance to improve performance. Lighthouse insights are a crucial part of a website's tech stack rating.
```

some of the phrasing felt a bit unusual/repetitive to me personally, so some optional suggestions for improvements if you're up for them :)

AndreasKubasa replied:
Hello @patrickhulce 

Wow! Thank you for your detailed report and suggestions.
Yes. I am the main maintainer of the project.

We will start to work in, your suggestions immediately. And thank you for finding the small things developers are mostly blind for when they are working on it -> 0-39 :)

We will rephrase some texts, as this is already a ticket on our Roadmap, we will do some necessary adaptions now.

You wrote:
>  "It'd be awesome to be able to actually open up the Lighthouse report that underlies the quality score"

Yes. There was a mistake in our documentation and "lack of transparency" I would say.
The Quality score comes from Best Practices, Accessibility and PWA.

Performance and SEO are characteristics for a good tech stack too, but the main fact that performance is related to the network makes it not suitable for our Algorythm.

Yes, that was one thing we were discussing about this week. First we wanted to add this functionality for users which have a login (which is not yet live). To store the reports in our database needs quite some space :). 
But we will show the lighthouse report on demand when clicking on "view report" which will be added soon, then we will send the report as html and render it on our website.

The awesomeness score is a combination of quality, powered by lighthouse and technology stats gathered all over open source community tools, like Github stars, Maintaners, pulse, last release, vulnerabilities, Npm downloads, jsdeliver stats etc..

I will update the pull request when our changes are live.

AndreasKubasa commented:
@patrickhulce We updated our websites.
Thank you once again for your suggestions.

We implemented the lighthouse report view -> https://awesometechstack.com/tools/analyze/website/reports/lighthouse/?url=awesometechstack.com

when clicking on:
![view-lighthouse](https://user-images.githubusercontent.com/55059012/77065026-cf50ef00-69e0-11ea-9d90-f7a782560fd3.PNG)

After implementing our user login we will add further reports and full reports for logged in users 👍 

Any further suggestions from you are very welcome :)

patrickhulce commented:
Amazing response time thanks for being so receptive @AndreasKubasa ! 

Found just 1 typo this time :)

> These values combined are used to calculate the abverage quality score of your website.

s/abverage/average/

re:

> Performance and SEO are characteristics for a good tech stack too, but the main fact that performance is related to the network makes it not suitable for our Algorythm.

We would definitely want to encourage you to consider factoring in the performance results in some way to your scoring. While the network definitely has a strong influence on the scale of performance numbers, the tech stack and decisions made in the architecture of the frontend can result in a 1s FCP or a 20s FCP, two radically different experiences (and 20s FCP would be really hard pressed to call an "awesome" tech stack 😉). As it stands currently, I think we can either remove all the references to performance in the description of this integration and proceed, or wait to factor in performance to the "quality" result and the Lighthouse report.

AndreasKubasa commented:
@patrickhulce 

> the tech stack and decisions made in the architecture of the frontend can result in a 1s FCP or a 20s FCP, two radically different experiences (and 20s FCP would be really hard pressed to call an "awesome" tech stack 😉).

💯 You are absolutely right. We have a ticket in our Roadmap which defines the weight per score, tech, quality and lighthouse quality score weights. 

We will implement performance and SEO lighthouse scores into our algorithm, as we are going to release update notes to our algorithm changes, our score is adaptable to it. Technologies and best practices are changing fast so we are also focusing on a reliable score but flexible in changes.

Thank you for your input. I appreciate it!

I will come back to you when finished and released. 👍

AndreasKubasa commented:
@patrickhulce Thank you for encouraging us to take the performance score into account.
We implemented and published the version using all lighthouse scores 👍 

Thank you for your patience.

AndreasKubasa commented:
@patrickhulce 

Thank you for your assitance and merge ;)

Just a quick followup question. Is there a chance to get mentioned in the npm's lighthouse integrations part  -> https://www.npmjs.com/package/lighthouse#lighthouse-integrations

patrickhulce commented:
> Just a quick followup question. Is there a chance to get mentioned in the npm's lighthouse integrations part -> https://www.npmjs.com/package/lighthouse#lighthouse-integrations

That happens automatically when master is published to npm. It will go out when 6.0 becomes final (it's currently in beta).

AndreasKubasa commented:
Sounds cool. Thank you!
